app/src/main/java/corp/kairos/adamastor/AppDetailComparator.java

```diff
@@ -1,10 +0,0 @@
-package corp.kairos.adamastor;
-
-import java.util.Comparator;
-
-public class AppDetailComparator implements Comparator<AppDetail> {
-    @Override
-    public int compare(AppDetail o1, AppDetail o2) {
-        return o1.getLabel().compareTo(o2.getLabel());
-    }
-}
```

Comment:
This class is redundant since the class `AppDetail` as already a method called `comparator`, and that's the best approach.
The comparator class should only be used when you have multiple comparators to a single class, and that's not the case here.
Short version: it's a refactor, it's all working, we don't need this class anymore.

Reply:
ok, you are right there 👍

Reply:
ok then